src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -442,7 +439,7 @@
             {% endif %}
 
             <span {% if has_error or data.has_children_error|default(false) %}class="has-error"{% endif %}>
-                {{ name|default('(no name)') }} {% if data.type_class is defined %}[<abbr title="{{ data.type_class }}">{{ data.type_class|split('\\')|last }}</abbr>]{% endif %}
+                {{ name|default('(no name)') }} [{{ profiler_dump(data.type_class) }}]
```

Comment:
as dumps are block elements and not inline, this will completely destroy the layout of the tree and page (same applies to line 460):

![sf-profiler-dumper-form-bug](https://cloud.githubusercontent.com/assets/749025/18225059/737b7d5e-71ea-11e6-8af3-53ac071b9b10.png)

We should either allow to dump inline (`<code>...</code>` instead of `<pre>...</pre>`) or we should revert this change